### ISSUE TITLE
Train of model fails on  tee() in method LabelStudioMLManager.train_script_wrapper() of   label-studio-ml-backend/model.py  

### ISSUE THREAD

**Issue by d3vopz-net** [heartexlabs/label-studio#3998]
**Describe the bug**
The tee() function expects an iterable object. but `task` is a list object. The `tee()` returns emtpy `datastream`. Following code fails.

**To Reproduce**

**Expected behavior**
tee() returns a tuple of iterations for datastream and snapshot.

**Screenshots**

**Environment (please complete the following information):**
 - Label Studio Version [1.7.2]

**Additional context**

The fix is to iter(tasks) `data_stream = iter(tasks)` instead simply assign task:
```

        if cls.without_redis():
            data_stream = iter(tasks)
        else:
            data_stream = (json.loads(t) for t in cls._redis.lrange(cls._get_tasks_key(project), 0, -1))

        if workdir:
            data_stream, snapshot = tee(data_stream)
            cls.create_data_snapshot(snapshot, workdir)

        try:
            train_output = m.model.fit(data_stream, workdir, **train_kwargs)
```

**Comment by d3vopz-net** [comment]
I used old ml-backend instance, not compatible with labelstudio 1.7.2. maybe helpfull to add some versions information in ml-backend and query from labelstudio to avoid. closing
